src/coreclr/jit/block.h

```diff
@@ -429,6 +429,7 @@ enum BasicBlockFlags : unsigned __int64
                                                           // (TODO: Remove this quirk after refactoring Compiler::fgFindInsertPoint)
     BBF_OLD_LOOP_HEADER_QUIRK          = MAKE_BBFLAG(42), // Block was the header ('entry') of a loop recognized by old loop finding
     BBF_HAS_VALUE_PROFILE              = MAKE_BBFLAG(43), // Block has a node that needs a value probing
+    BBF_JMP_TO_NESTED_LOOP             = MAKE_BBFLAG(44),
```

Comment:
Please add a comment

Reply:
I plan to remove this flag before merging; this was just a quick-and-dirty way for determining when to remove a jump in a `BBJ_ALWAYS` block with alignment.